tests/js/spec/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -1050,38 +1049,6 @@ describe('WidgetBuilder', function () {
     });
   });
 
-  it('disables dashboards with max widgets', async function () {
```

Comment:
Deleted this test because there is no longer a "Select Dashboard" step in the v2 design